### ISSUE TITLE
[Brave News]: Accessibility DCHECK failure on MacOS

### ISSUE THREAD

**Issue by fallaciousreasoning** [brave/brave-browser#42054]
https://bravesoftware.slack.com/archives/C3T9S9WKD/p1730409235891129?thread_ts=1730407022.108129&cid=C3T9S9WKD
